pkg/trace/event/processor.go

```diff
@@ -66,24 +67,26 @@ func (p *Processor) Process(root *pb.Span, t *pb.TraceChunk) (numEvents, numExtr
 		}
 
 		numExtracted++
-
-		sampled, epsRate := p.maxEPSSample(span, priority)
-		if !sampled {
-			continue
+		//event analytics tags shouldn't be set on sampled single spans
```

Comment:
```suggestion
		// event analytics tags shouldn't be set on sampled single spans
```
Can we put this inside the `if` statement?